Sources/AsyncQueue/ActorQueue.swift

```diff
@@ -23,53 +23,52 @@
 /// A queue that executes asynchronous tasks enqueued from a nonisolated context.
 /// Tasks are guaranteed to begin executing in the order in which they are enqueued. However, if a task suspends it will allow subsequently enqueued tasks to begin executing.
 /// Asynchronous tasks sent to this queue execute as they would in an `actor` type, allowing for re-entrancy and non-FIFO behavior when an individual task suspends.
+/// Tasks execute within the target actor's isolated context
 ///
 /// An `ActorQueue` is used to ensure tasks sent from a nonisolated context to a single `actor`'s isolated context begin execution in order.
 /// Here is an example of how an `ActorQueue` should be utilized within an `actor`:
 /// ```swift
 /// public actor LogStore {
 ///
+///     public init() {
+///         queue.setTargetContext(to: self)
```

Comment:
I don't love the `setTargetContext(to:)` method signature. I welcome better suggestions!